integration/apps/nascg/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#  Copyright (c) 2015 - 2022, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+set -x
+set -e
+
+# Get helper functions
+source ../build_func.sh
+
+# Set variables for workload
+DIRNAME=NPB3.4.2
+ARCHIVE=NPB3.4.2.tar.gz
+URL=https://www.nas.nasa.gov/assets/npb
+
+# Run helper functions
+clean_source ${DIRNAME}
+get_archive ${ARCHIVE} ${URL}
+unpack_archive ${ARCHIVE}
+setup_source_git ${DIRNAME}
+
+# Build application
+cd ${DIRNAME}/NPB3.4-MPI
+make suite
```

Comment:
All these new apps compile fine on borealis.

Reply:
This is an mpif90 problem on mcfly.  If you want to build there, you need to modify the NPB build to use mpiifort instead of mpif90.  I tested and confirmed that solution. 
 Related thread:
https://community.intel.com/t5/Intel-MPI-Library/Intel-MPI-Fortran-2008-Error/m-p/1360122

Since the preferred solution is to stay with mpif90 here, if we ever need to build on mcfly we'd have to create a build variant for it that applies an extra patch to switch out the compiler.